static/js/reminder.js

```diff
@@ -96,7 +99,11 @@ export function schedule_message(request = compose.create_message_object()) {
     };
     const error = function (response) {
         $("#compose-textarea").prop("disabled", false);
-        compose_error.show(_.escape(response), $("#compose-textarea"));
+        compose_error.show_error_message(
+            response,
+            compose_error.CLASSNAMES.generic_error,
```

Comment:
This also appears broken again.

Reply:
Oh sorry! Not part of my usual workflow. I'm also happy to add the fixes back if that's easier for you.

Reply:
Nah, I've already pulled it down and done the rebase.